examples/faith/pegasus_workflow_conf.ini

```diff
@@ -0,0 +1,137 @@
+[lalapps_inspinj]
+min-mass1 = 50
+max-mass1 = 150
+min-spin1 = 0
+max-spin1 = 1
+min-mass2 = 50
+max-mass2 = 150
+min-spin2 = 0
+max-spin2 = 1
+min-mtotal = 100
+max-mtotal = 300
+enable-spin =
+aligned =
+f-lower = 20
+waveform = IMRPhenomXAS
+disable-milkyway =
+i-distr = uniform
+l-distr = random
+min-distance = 1000
+d-distr = uniform
+max-distance = 1000
+gps-start-time = 1000000000
+gps-end-time = 1000001000
+time-interval = 0.
+time-step = 1.
+seed = 123434
+m-distr = componentMass
+
+[executables]
+lalapps_inspinj = ${which:lalapps_inspinj}
+pycbc_faithsim = ${which:pycbc_faithsim}
+pycbc_faithsim_collect_results = ${which:pycbc_faithsim_collect_results}
+pycbc_faithsim_plot = ${which:pycbc_faithsim_plots}
+pycbc_splitbank = ${which:pycbc_splitbank}
+
+[workflow]
+log-path = /home/marion.pillas/projects/template_bank/log_file/
+file-retention-level = all_triggers
+
+[splitbank]
+num_banks = 10
+
+[pycbc_faithsim]
+psd-model = aLIGOZeroDetHighPower
+waveform1-approximant = IMRPhenomXAS
+waveform1-start-frequency = 14
+waveform2-approximant = SEOBNRv4_opt
+waveform2-start-frequency = 14
+filter-low-frequency = 20
+filter-sample-rate = 16384
+filter-waveform-length = 1024
+
+[pycbc_faithsim_plots_mass1_mass2_time_offset]
```

Comment:
This should really be `pycbc_faithsim_plots-mass1_mass2_time_offset` (note the '-' between the exe name and the extra bit) to match the standard naming convention. Then you can just use tags to let the workflow generator pick up the options automatically.